portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,11 +26,19 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-    private String name = "Roland";
+    ArrayList<String> user = new ArrayList<>(
+            Arrays.asList("Roland","Naijuka","03/20","University of Rochester","Computer Science")
+        );
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("text/html;");
-        response.getWriter().println("<h1>Hello " + name +"!</h1>");
+        String userInJson = convertToJson(user);
+        response.setContentType("application/json;");
+        response.getWriter().println(userInJson);
+    }
+
+    private String convertToJson(ArrayList<String> user){
+        Gson gson  = new Gson();
```

Comment:
Does this need to be initialized every time, or can it be initialized once and reused?

Reply:
> Does this need to be initialized every time, or can it be initialized once and reused?

It is actually used once, I am gonna initialize it where it is used